src/business_logic/execution/execution_entry_point.rs

```diff
@@ -55,6 +41,8 @@ pub(crate) struct ExecutionEntryPoint {
 }
 
 impl ExecutionEntryPoint {
+    // TODO: Remove warning inhibitor when finally used.
+    #[allow(dead_code)]
```

Comment:
Needed?

Reply:
Points to the same line as the previous comment. Marking as resolved since it's duplicated.